projects/lib/src/lib/pipes.ts

```diff
@@ -111,6 +131,16 @@ export class GetInProgressPipe implements PipeTransform {
   }
 }
 
+@Pipe({ name: 'successOrProgressValue' })
```

Comment:
Please, rename to `successOrInProgressValue` and `GetSuccessOrInProgressValuePipe`. For consistency.

Reply:
ok